bin/install-qa-check.d/60pkgconfig

```diff
@@ -84,7 +84,8 @@ pkgconfig_check() {
 			fi
 		elif [[ ${f} == *lib64/pkgconfig* ]] ; then
 			# We want to match /lib/, /lib/foo/, but not e.g. /lib64 or /lib64/, or libfoo
-			if grep -E -q '=(/usr)?/lib\b' ${f} ; then
+			# Ignore compiler= lines which can refer to compiler wrappers under /usr/lib/ccache for example
+			if grep -E -q '^libdir=(/usr)?/lib\b' "${f}" ; then
```

Comment:
Yeah, libdir seems right for sure.

Maybe also check for "-L.../lib" as well. Some examples from my system:
```
/usr/lib64/pkgconfig/mujs.pc:Libs: -L/usr/lib -lmujs
/usr/lib64/pkgconfig/nspr.pc:Libs: -L/usr/lib64 -lplds4 -lplc4 -lnspr4
/usr/lib64/pkgconfig/KF6Attica.pc:Libs: -L${prefix}/lib64 -lKF6Attica
```

Reply:
You've discovered a hardcoded libdir in the mujs Makefile:
```make
libdir = $(prefix)/lib
```

Reply:
Included both Libs: and Libs.private: liine.

Reply:
Looks like the mujs libdir issue has been reported already in [bug 963883](https://bugs.gentoo.org/963883).